interface/super/edit_list.php

```diff
@@ -1335,6 +1422,16 @@ class="btn btn-default btn-save"><?php echo xlt('Save'); ?></button>
             DeleteList(this);
         });
 
+        //prevent Enter button press on filter
```

Comment:
As we begin the process of modernizing our JS code, we need to stop relying on jquery for tasks that can be accomplished with regular javascript.